src/styles/_thermostat.scss

```diff
@@ -397,7 +397,152 @@
   }
 }
 
+@mixin thermostat-detail-section {
```

Comment:
Last step would be removing `thermostat` from the style namespace.

Reply:
Oh good catch—thanks